### ISSUE TITLE
[Suggestion] Config for differentiating growth time between Hopping/Non-Hopping Pots 

### ISSUE THREAD

**Issue by NielsPilgaard** [Darkhax-Minecraft/BotanyPots#136]
Hi there!

Would you consider making a config option or datapack field for differentiating growth time between Hopping and Non-Hopping Botany Pots? I'm looking to nerf Botany Pots a bit, but don't see the need to nerf the non-hopping variety quite as much as the hopping.

**Comment by CodexHere** [comment]
I would like some kind of config as well for growth time. It becomes incredibly OP, way too quick, and very capable early game.

We have new players that are able to essentially get infinite food within 20 minutes of joining the server for the first time, which just seems like a waste.

**Comment by Darkhax** [comment]
@NillerMedDild I will consider this for a future update. 

@nerdfoundrygaming This is already possible with something like CraftTweaker. For example this script can be used to make every crop 20x slower. 

```zs
import mods.botanypots.ZenCrop;
val crops = <recipetype:botanypots:crop>;

crops.forEach((recipe) => {
    
    recipe.setGrowthTicks(recipe.getGrowthTicks() * 20);
});
```

**Comment by fpbrault** [comment]
> @NillerMedDild I will consider this for a future update.
> 
> @nerdfoundrygaming This is already possible with something like CraftTweaker. For example this script can be used to make every crop 20x slower.
> 
> ```zenscript
> import mods.botanypots.ZenCrop;
> val crops = <recipetype:botanypots:crop>;
> 
> crops.forEach((recipe) => {
>     
>     recipe.setGrowthTicks(recipe.getGrowthTicks() * 20);
> });
> ```

This script does not appear to work on my 1.16.5 server, as I get this error: `growth.zs:4:0 No such member: forEach`

Is this normal?

**Comment by slebreton79** [comment]
Yeah this example script I get the same error... no such member forEach

**Comment by Darkhax** [comment]
Yes, the example script does not work. It looks like that version was never publicly released. I will look into getting that version out soon.

**Comment by CodexHere** [comment]
I personally wasn't wanting to add CraftTweaker to the environment, so I opted for a datapack that just updates the recipes.

There wasn't any sophistication done here - every item takes 30min to mature with this pack.
I originally planned on scalling them all linearly based on their pre-packaged settings, but when it came down to it I wanted crops to take a bit to mature to avoid abuse, and it was easy to string-replace across the entire set of recipes.

Feel free to use this in the meantime, tweak as you see fit, etc... It's nothing special, just already done.

[botanypots_vanillarize.zip](https://github.com/Darkhax-Minecraft/BotanyPots/files/6274802/botanypots_vanillarize.zip)
